### PR TITLE
Added function addTooltip, fixed bugs #1594,#1240

### PR DESCRIPTION
**Pull request by christykobyletska**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/pages/Club/Club/ClubFollowers.tsx [hunk 1]
```diff
@@ -10,7 +10,9 @@ import Title from 'antd/lib/typography/Title';
 import Spinner from '../../Spinner/Spinner';
 import NotificationBoxApi from '../../../api/NotificationBoxApi';
 import { Modal } from 'antd';
+import addTooltip from '../../../components/Tooltip';
 
+const namesMaxLegth = 21;
```
Comment (ostappppppp) [inline src/pages/Club/Club/ClubFollowers.tsx:15]: this `21` value appears multiple times
so I guess better option is to declare it right in `addTooltip` component

**Review by ostappppppp — CHANGES_REQUESTED** [review]

**Review by solomiyon — APPROVED** [review]

**Review by nicoolle — APPROVED** [review]

File: src/components/Tooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { Tooltip } from "antd";
+import React from "react";
+
+function addTooltip (number: number, text: string) {
```
[the remark this answers is not in the document]
Comment (christykobyletska) [reply src/components/Tooltip.tsx:4]: what do you mean? don't understand this

**Review by sharix4 — APPROVED** [review]

File: src/pages/City/City/CityMembers.tsx [hunk 3]
```diff
@@ -14,8 +14,10 @@ import Title from "antd/lib/typography/Title";
 import Spinner from "../../Spinner/Spinner";
 import NotificationBoxApi from "../../../api/NotificationBoxApi";
 import { Roles } from "../../../models/Roles/Roles";
+import addTooltip from "../../../components/Tooltip";
 moment.locale("uk-ua");
 
+const namesMaxLegth = 21;
```
[the remark this answers is not in the document]
Comment (christykobyletska) [reply src/pages/City/City/CityMembers.tsx]: delete at all

File: src/pages/Club/Club/ClubFollowers.tsx (hunk 1 above)
Comment (christykobyletska) [reply src/pages/Club/Club/ClubFollowers.tsx]: done

File: src/components/Tooltip.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (christykobyletska) [reply src/components/Tooltip.tsx]: done

**Review by ostappppppp — APPROVED** [review]
LGTM

**Pull request merged**